proto/aserto/directory/model/v3/model.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package aserto.directory.model.v3;
+
+option go_package = "github.com/aserto-dev/go-directory/aserto/directory/model/v3;model"; 
+
+import "google/protobuf/empty.proto";
+
+import "aserto/directory/common/v3/common.proto";
+
+service Model {
```

Comment:
I thought we wanted the model API to stream raw bytes.
@oanatmaria has a draft PR with a proposal.

Reply:
Agreed, as discussed this morning, same for the schema_version being part of the manifest payload, so that should get removed.